exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/FragmentWrapper.java

```diff
@@ -136,26 +240,60 @@ public String getContent() {
         biggestBatches = Math.max(biggestBatches, batches);
       }
 
-      builder.appendCell(new OperatorPathBuilder().setMajor(major).setMinor(minor).build(), null);
-      builder.appendCell(minor.getEndpoint().getAddress(), null);
-      builder.appendMillis(minor.getStartTime() - start, null);
-      builder.appendMillis(minor.getEndTime() - start, null);
-      builder.appendMillis(minor.getEndTime() - minor.getStartTime(), null);
+      builder.appendCell(new OperatorPathBuilder().setMajor(major).setMinor(minor).build());
+      builder.appendCell(minor.getEndpoint().getAddress());
+      builder.appendMillis(minor.getStartTime() - start);
+      builder.appendMillis(minor.getEndTime() - start);
+      builder.appendMillis(minor.getEndTime() - minor.getStartTime());
 
-      builder.appendFormattedInteger(biggestIncomingRecords, null);
-      builder.appendFormattedInteger(biggestBatches, null);
+      builder.appendFormattedInteger(biggestIncomingRecords);
+      builder.appendFormattedInteger(biggestBatches);
 
-      builder.appendTime(minor.getLastUpdate(), null);
-      builder.appendTime(minor.getLastProgress(), null);
+      builder.appendTime(minor.getLastUpdate());
+      builder.appendTime(minor.getLastProgress());
 
-      builder.appendBytes(minor.getMaxMemoryUsed(), null);
-      builder.appendCell(minor.getState().name(), null);
+      builder.appendBytes(minor.getMaxMemoryUsed());
+      builder.appendCell(minor.getState().name());
     }
 
     for (final MinorFragmentProfile m : incomplete) {
-      builder.appendCell(major.getMajorFragmentId() + "-" + m.getMinorFragmentId(), null);
+      builder.appendCell(major.getMajorFragmentId() + "-" + m.getMinorFragmentId());
       builder.appendRepeated(m.getState().toString(), null, NUM_NULLABLE_FRAGMENTS_COLUMNS);
     }
     return builder.build();
   }
+
+  private class OverviewTblTxt {
+    static final String MajorFragment = "Major Fragment";
+    static final String MinorFragmentsReporting = "Minor Fragments Reporting";
+    static final String FirstStart = "First Start";
+    static final String LastStart = "Last Start";
+    static final String FirstEnd = "First End";
+    static final String LastEnd = "Last End";
+    static final String MinRuntime = "Min Runtime";
+    static final String AvgRuntime = "Avg Runtime";
+    static final String MaxRuntime = "Max Runtime";
+    static final String PercentBusy = "% Busy";
+    static final String LastUpdate = "Last Update";
+    static final String LastProgress = "Last Progress";
+    static final String MaxPeakMemory = "Max Peak Memory";
+  }
+
+  private class OverviewTblTooltip {
+    static final String MajorFragment = "Major Fragment ID";
+    static final String MinorFragmentsReporting = "Number of Minor Fragments Spawned";
```

Comment:
This is descriptive text, so use sentence case: "Number of minor fragments spawned". Maybe "spawened" --> "started".